pre-processing/primod/driver_coupling/ribameta.py

```diff
@@ -28,8 +29,8 @@ class RibaMetaDriverCoupling(DriverCoupling):
         GeoDataFrame of user demand polygons
     """
 
-    ribasim_basin_definition: gpd.GeoDataFrame
-    ribasim_user_demand_definition: gpd.GeoDataFrame | None = None
+    ribasim_basin_definition: gpd.GeoDataFrame | xr.DataArray
```

Comment:
Should be `gpd.GeoDataFrame | xr.Dataset` right, not DataArray?
With the variable names matching the MF6 keys.

Reply:
For MetaSWAP coupling we need a single basin definition. A ``` xr.DataArray ``` would be sufficient right?